pureelk.sh

```diff
@@ -175,9 +179,31 @@ start_containers() {
   else
       print_warn "$PUREELK is already running."
   fi
+  
+  print_info "Start Logrotate container..."
+  RUNNING="$(docker inspect -f '{{.State.Running}}' $LOGROTATE)"
+  if [ $? -eq 1 ];
+  then
+      print_warn "$LOGROTATE doesn't exist, starting..."
+      docker run -d -v /var/lib/docker/containers:/var/lib/docker/containers -v /var/log/pureelk:/var/log/pureelk --name $LOGROTATE \
+        -e "LOGS_DIRECTORIES=/var/lib/docker/containers /var/log/pureelk" \
```

Comment:
We don't need to set it. Default is 5. This is the documentation:
https://github.com/blacklabelops/logrotate/blob/master/README.md

I have tested in my local with the output in the other thread.